cli/slsa-verifier/verify/verify_artifact.go

```diff
@@ -39,40 +39,52 @@ type VerifyArtifactCommand struct {
 	PrintProvenance     bool
 }
 
-func (c *VerifyArtifactCommand) Exec(ctx context.Context, artifacts []string) (*utils.TrustedBuilderID, error) {
-	artifactHash, err := getArtifactHash(artifacts[0])
-	if err != nil {
-		return nil, err
-	}
+func (c *VerifyArtifactCommand) Exec(ctx context.Context, artifacts []string) ([]*utils.TrustedBuilderID, error) {
+	var builderIds []*utils.TrustedBuilderID
 
-	provenanceOpts := &options.ProvenanceOpts{
-		ExpectedSourceURI:      c.SourceURI,
-		ExpectedBranch:         c.SourceBranch,
-		ExpectedDigest:         artifactHash,
-		ExpectedVersionedTag:   c.SourceVersionTag,
-		ExpectedTag:            c.SourceTag,
-		ExpectedWorkflowInputs: c.BuildWorkflowInputs,
-	}
+	for _, artifact := range artifacts {
+		fmt.Fprintf(os.Stderr, "Verifying artifact %s\n", artifact)
```

Comment:
Since this is printed below for each artifact in the SUCCESS/FAILED per artifact, should we remove this? Ideally (and I think you did) catch all return points

Reply:
Sure. Removed it. I initially included it to have a bracket-type output

```
starting action
  details
  mode details
finished action: result
```

But then didn't add the indentation to the other lines, so not as relevant. Maybe we can customize output at another time.